libbeat/outputs/elasticsearch/api_test.go

```diff
@@ -134,3 +136,12 @@ func TestReadSearchResult_invalid(t *testing.T) {
 func newTestClient(url string) *Client {
 	return newTestClientAuth(url, "", "")
 }
+
+func (r QueryResult) String() string {
```

Comment:
yup. Somewhat ugly to add a method for testing though. But it's some old code I didn't want to clean up as well.